truesilver-azshara-spawn-guid-9329-update.sql

```diff
@@ -0,0 +1 @@
+UPDATE `gameobject` SET `position_y` = -5124.500500, `position_z` = 87.242325 WHERE `guid` = 9329;

```

Comment:
What Bench said, but also you need, when UPDATEing the gameobject or creature table, also specify the id of the gameobject in question.
```sql
WHERE `guid` = X AND `id` = Y;
```